Api/Options/ApplicationOptions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+using Diagnosea.Submarine.Api.Abstractions.Settings;
+
+namespace Diagnosea.Submarine.Api.Options
+{
+    public class ApplicationOptions : ISubmarineAuthenticationSettings
```

Comment:
Guess we can rename this to `AuthenticationSettings` or `SubmarineAuthenticationSettings`.